x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -372,6 +373,29 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.anomaliesTable.assertTableRowsCount(10);
         });
 
+        it('renders swim lanes correctly on the time bounds change', async () => {
+          const fromTime = 'Jul 7, 2012 @ 00:00:00.000';
+          const toTime = 'Feb 12, 2016 @ 23:59:54.000';
+
+          await PageObjects.timePicker.pauseAutoRefresh();
+          await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+
+          await ml.commonUI.waitForDatePickerIndicatorLoaded();
+
+          await ml.swimLane.waitForSwimLanesToLoad();
+          await ml.swimLane.assertAxisLabels(viewBySwimLaneTestSubj, 'x', [
```

Comment:
Should we add retries to this assertAxisLabels?

Reply:
exactly @qn895 , I'm having hard time with `PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);`, for some reason it gets stuck in the loop. I assumed there are some issue with the tests running locally, but it also happens in CI